imap/annotate.c

```diff
@@ -2725,15 +2501,18 @@ EXPORTED int annotate_state_fetch(annotate_state_t *state,
          */
         g = glob_init(s, '.');
 
-        for (attribcount = 0;
-             annotation_attributes[attribcount].name;
-             attribcount++) {
+        for (attribcount = 0; annotation_attributes[attribcount].name;
+             attribcount++)
+        {
```

Comment:
Hmm, I don't like what it did here, I'd prefer each statement on its own line if they don't all fit on one.  But I didn't see an option to control this behaviour.